### ISSUE TITLE
NTR, pulmo: Air trapping on Expiration

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5581]
**Preferred term label:**
Air trapping on Expiration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal retention of gas within a lung or part of a lung, as a result of airway obstuction of abnormalities in lung compliance.  In the classic presentation, the lung will appear normal at inspiration, but on exhalation, the diseased portions of the lung which have lost connective tissue recoil will remain lucent while the healthy portions of the lung will become more dense due to atelectasis. This helps distinguish it from mosaic attenuation due to patchy fibrosis, as occurs with nonspecific interstitial pneumonia, and in early usual interstitial pneumonitis (the hallmark imaging diagnosis of interstitial lung disease) in which there is no change with inspiration and expiration. 

Comment: The extent of air trapping present on expiratoryscans can be measured using a semiquantitative scoring system, which estimates the percentage of lung that appears abnormal on each scan. Such systems have the advantage of being simple, quick, and easy to perform at the time of image interpretation. Furthermore, in one study, a simple 5-point scoring system was found to be associated with better interobserver agreement than a more detailed scoring system

PMID: 18195376, PMID: 10524809, https://doi.org/10.1148/radiology.193.2.7972745

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
emphysema (357,364,365), chronic airways disease (359), cystic fibrosis (ORPHA:586) Hypersensitivity Pneumonitis (OMIM 145300, ORPHA:31740), Langerhans histiocytosis (ORPHA:389) and tuberous sclerosis (ORPHA:805), bronchiectasis (ORPHA:60033), Hypersensitivity Pneumonitis (OMIM 145300, ORPHA:31740), sarcoidosis (ORPHA:797). 

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
For consistency, I will use the label Expiratory air trapping and have Air trapping on Expiration be a synonym

**Comment by pnrobinson** [comment]
adding term as HP:0032968
